grakn-client/src/main/java/ai/grakn/client/TaskClient.java

```diff
@@ -111,7 +111,7 @@ public TaskId sendTask(Class<?> taskClass, String creator, Instant runAt, Durati
 
             return TaskId.of(jsonResponse.at("id").asString());
         } catch (IOException e){
-            throw new EngineUnavailableException(e);
+            throw GraknBackendException.engineUnavailable(host, port, e);
```

Comment:
This PR might be a good opportunity to add the `throws` declarations to the functions when appropriate.